src/test/scala/org/apache/spark/streaming/rabbitmq/RabbitMQConsumer.scala

```diff
@@ -36,35 +36,29 @@ object RabbitMQConsumer {
     // options, along with the default values.
     // All the parameters are shown below, remove the ones
     // that you don't need
-    val receiverStream = RabbitMQUtils.createStream(ssc, Map(
-      "host" -> "localhost",
+    val receiverStream = RabbitMQUtils.createStream[String](ssc, Map(
+      "hosts" -> "localhost",
       "queueName" -> "rabbitmq-queue",
       "exchangeName" -> "rabbitmq-exchange",
-      "vHost" -> "rabbitmq-vHost",
-      "username" -> "rabbitmq-user",
-      "password" -> "rabbitmq-password",
-      "x-max-length" -> "value",
-      "x-max-length" -> "value",
-      "x-message-ttl" -> "value",
-      "x-expires" -> "value",
-      "x-max-length-bytes" -> "value",
-      "x-dead-letter-exchange" -> "value",
-      "x-dead-letter-routing-key" -> "value",
-      "x-max-priority" -> "value"
+      "vHost" -> "/",
+      "username" -> "guest",
+      "password" -> "guest"
```

Comment:
It's good to have all the parameters as an example, but it could be ok like this